src/OneExplorer/ConfigObject.ts

```diff
@@ -321,26 +334,26 @@ export class ConfigObj {
     if (artifacts.length > 1) {
       // TODO Notify the error with a better UX
       // EX. put question mark next to the config icon
-      Logger.warn('OneExplorer', `There are multiple input models in the configuration(${uri}).`);
+      Logger.warn(
+          'OneExplorer', `There are multiple input models in the configuration(${filePath}).`);
     }
-
     if (artifacts.length === 0) {
       // TODO Notify the error with a better UX
       // EX. showing orphan nodes somewhere
-      Logger.warn('OneExplorer', `There is no input model in the configuration(${uri}).`);
+      Logger.warn('OneExplorer', `There is no input model in the configuration(${filePath}).`);
     }
 
     // Return as list of uri
-    return artifacts.map(artifact => vscode.Uri.file(artifact.path!));
+    return artifacts;
   };
 
   /**
    * Find derived models written in the ini object and return the absolute path.
    *
    * @param uri cfg uri is required to calculate absolute path
    */
-  private static parseDerivedModels = (uri: vscode.Uri, iniObj: object): vscode.Uri[] => {
-    const dir = path.dirname(uri.fsPath);
+  private static parseDerivedModels = (filePath: string, iniObj: object): Artifact[] => {
```

Comment:
(5) Pass string filepath instead of Uri as parameter of functions (parseDerivedModels, parseBaseModels)

There is no specific reason for the function to get it as Uri.
In a while, this change removes redundant translation from string to uri.